Storage.md

```diff
@@ -0,0 +1,261 @@
+```
+
+## Updating
+
+When an item's value changes, the module checks its triggers, modularity, and element status.
+
+Triggers are functions that get run when an item's value equals the specified value.
+
+```typescript
+ItemsHolder.addItem("color", {
+    triggers: {
+        red: () => { this.value = "black"; }
+    }
+});
+ItemsHolder.setItem("color", red);
+```
+
+Modularity restricts an item's value to a range from 0 to the specified max and runs the `onModular` function as many times until the value hits the modular range.
+
+```typescript
+ItemsHolder.addItem("time", {
+    modularity: 12,
+    num: 0,
+    onModular: () => { this.num += 1; }
+});
+ItemsHolder.setItem("time", 100);
+```
+
+If the item is an HTML element, the value is also updated on the page.
+
+```typescript
+ItemsHolder.addItem("color", {
+    hasElement: true,
+    element: {}
+});
+ItemsHolder.setItem("color", "red");
+```
+
+### Auto Save
+
+By default, the values in localStorage are not updated when their values change.
+Auto saving which updates localStorage values can be enabled when making the ItemsHolder container.
+
+```typescript
+let ItemsHolder = new ItemsHoldr({ autoSave: true });
+```
+
+To toggle the autoSave feature, use `toggleAutoSave`.
+
+```typescript
+ItemsHolder.toggleAutoSave();
+```
+
+## Clearing and Defaults
+
+To clear all items from the collections, use `clear`.
+
+```typescript
+ItemsHolder.clear();
+```
+
+To clear the collection, but keep some items which will always be in the collection, utilize ItemHoldr's `valueDefault` property.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({
+    values: {
+        color: {
+            valueDefault: "red"
+        }
+    }
+});
+ItemsHolder.clear(); // "color" is still in the collection with a reset value of "red"
+```
+
+## Elements
+
+HTML elements stored within ItemsHoldr will have their values updated on the page when changed.
+To signal if a container should be made to hold HTML elements, assign a value to `doMakeContainer`.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({ doMakeContainer: true });
+```
+
+With this, any HTML items in the collection at the time of construction are held in ItemHolder's `container` object.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({
+    doMakeContainer: true,
+    values: {
+        color: {
+            valueDefault: "red",
+            hasElement: true,
+            element: {}
+        }
+    }
+});
+
+// color is in the container, but speed is not
+ItemsHolder.addItem("speed", {
+    valueDefault: 10,
+    haselement: true,
+    element: {}
+});
+```
+
+### Display Changes
+
+If you want certain values to be represented differently on the page, make use of `displayChanges`.
+These changes are hardcoded values that replace specified values when element items are updated.
+
+```typescript
+let ItemsHolder = new ItemsHoldr({
+    displayChanges: {
+        Infinity: "∞";
```

Comment:
You'll want to switch this to `"INF"` or similar. Some people have problems with unusual character encodings.